src/codelabs/iterables.md

```diff
@@ -972,12 +970,12 @@ which has a constructor that takes a string.
 Another provided function is `isValidEmailAddress()`,
 which tests whether an email address is valid.
 
-|----------------------+-------------------+----------------------------------|
-| Constructor/function | Type signature    | Description                       |
-|----------------------|-------------------|----------------------------------|
-| EmailAddress() | `EmailAddress(String address)` | Creates an `EmailAddress` for the specified address. | 
+|-----------------------+------------------------------------------+---------------------------------------------------------|
```

Comment:
I know formatting this isn't necessary, but I find it easier to read and edit this way.